Find-and-Replace/Replace-line-break-with-paragraph-mark/.NET-Standard/Replace-line-break-with-paragraph-mark/Program.cs

```diff
@@ -0,0 +1,147 @@
+            { 
+                //Check the text body items.
+                IEntity bodyItemEntity = textBody.ChildEntities[i];
+                //Check the entity type.
+                switch (bodyItemEntity.EntityType)
+                {
+                    case EntityType.Paragraph:
+                        WParagraph paragraph = bodyItemEntity as WParagraph;
+                        //Iterate through the paragraph.
+                        IterateParagraph(paragraph.Items);
+                        break;
+                    case EntityType.Table:
+                        //Iterate through table.
+                        IterateTable(bodyItemEntity as WTable);
+                        break;
+                    case EntityType.BlockContentControl:
+                        BlockContentControl blockContentControl = bodyItemEntity as BlockContentControl;
+                        //Iterate to the body items of Block Content Control.
+                        IterateTextBody(blockContentControl.TextBody);
+                        break;
+                }
+            }
+        }
+        /// <summary>
+        /// Iterate table child elements.
+        /// </summary>
+        private static void IterateTable(WTable table)
+        {
+            //Iterate the row collection in a table.
+            foreach (WTableRow row in table.Rows)
+            {
+                //Iterate the cell collection in a table row.
+                foreach (WTableCell cell in row.Cells)
+                {
+                    IterateTextBody(cell);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Iterate paragraph child elements.
+        /// </summary>
+        private static void IterateParagraph(ParagraphItemCollection paraItems)
+        {
+            for (int i = 0; i < paraItems.Count; i++)
+            {
+                Entity entity = paraItems[i];
+                //Check the element type by using EntityType.
+                switch (entity.EntityType)
+                {
+                    case EntityType.Break:
+                        Break breakItem = entity as Break;
+                        //Replace line break with paragraph mark.
+                        if (breakItem.BreakType == BreakType.LineBreak)
+                        {
+                            WParagraph ownerPara = breakItem.OwnerParagraph;
+                            int breakIndex = ownerPara.ChildEntities.IndexOf(breakItem);
+                            int paraIndex = ownerPara.OwnerTextBody.ChildEntities.IndexOf(ownerPara);
+
+                            //Create new paragraph by cloning the existing paragraph.
+                            WParagraph newPara = ownerPara.Clone() as WParagraph;
+                            //Remove the child items after the line break from the old paragraph including line break.
+                            for (int j = breakIndex; j < ownerPara.ChildEntities.Count;)
```

Comment:
If we not incrementing the index, then why we need to use for loop here?
Can you check and use the valid loop.